models/User.php

```diff
@@ -150,6 +150,8 @@ public function attemptActivation($code)
         {
             $message->to($this->email, $this->name);
         });
+
+        return true;
```

Comment:
previously returned null, therefore causing Account.php [line 201] to trow exception